shared-module/displayio/ColorConverter.c

```diff
@@ -128,9 +128,24 @@ bool common_hal_displayio_colorconverter_get_dither(displayio_colorconverter_t*
     return self->dither;
 }
 
+void common_hal_displayio_colorconverter_make_transparent(displayio_colorconverter_t* self, uint32_t transparent_color) {
+    self->transparent_color = transparent_color;
+}
+
+void common_hal_displayio_colorconverter_make_opaque(displayio_colorconverter_t* self, uint32_t transparent_color) {
+    if (self->transparent_color == transparent_color) {
+        m_del(uint8_t, self, transparent_color);
```

Comment:
The color is a simple value without it's own memory so don't call `m_del` on it. Instead, you'll either want a sentinel value that means the color is invalid, such as 0x1000000, which is higher than 24 bits of color. Or, you can use a separate boolean to track when the color is valid.

I think you'll want to raise an exception (`mp_raise...`) here if the given color doesn't match the existing one.